app/src/features/surveys/observations/sampling-sites/SamplingSiteList.tsx

```diff
@@ -49,6 +49,7 @@ const SamplingSiteList = () => {
 
   const [anchorEl, setAnchorEl] = useState<MenuProps['anchorEl']>(null);
   const [selectedSampleSiteId, setSelectedSampleSiteId] = useState<number | undefined>();
+  const [checked, setChecked] = useState<number[]>([]);
```

Comment:
Nit: might rename this to `checkboxSelectedIds` or `checkboxSelectedSampleSites`